components/alert/AlertPopulations.vue

```diff
@@ -56,17 +56,49 @@
 
       <div v-show="mode === 'number'">
         <line-chart
+          v-if="alert"
```

Comment:
This is still incorrect. You need to check if the chart actually has any data.

Reply:
Is this dataCollection?